hazelcast/src/main/java/com/hazelcast/client/impl/protocol/DefaultMessageTaskFactoryProvider.java

```diff
@@ -873,9 +873,9 @@
 public class DefaultMessageTaskFactoryProvider implements MessageTaskFactoryProvider {
     private static final int MESSAGE_TASK_PROVIDER_INITIAL_CAPACITY = 500;
 
-    private final Int2ObjectHashMap<MessageTaskFactory> factories;
+    protected final Int2ObjectHashMap<MessageTaskFactory> factories;
```

Comment:
nit: since this class is no longer extended on EE side, can revert these changes

Reply:
Agree, will revert tomorrow.

Reply:
Done for both fields.